### ISSUE TITLE
Losing preference settings

### ISSUE THREAD

**Issue by vkichline** [ropg/M5ez#73]
Hello, Rop!
I've been enjoying using M5ez a lot, but I had found the preferences seemed to be unreliable, except for the WiFi networks, which were always rock solid. I believe I tracked this down today.
In M5ez.cpp, in ```ezWifi::writeFlash()```, line 1211 is: ``` prefs.clear() ```.
I suspect this is a holdover from early days where networks were the only persistent settings. Now, however, this removes my other preferences.
I think what's intended is to specifically erase the networks. I have substituted the following for the beginning of the method, and seems to have solved my preferences issues.

```
    void ezWifi::writeFlash() {
        Preferences prefs;
        String idx;
        uint8_t n = 1;
        prefs.begin("M5ez", false);
        prefs.remove("autoconnect_on");
        while (true) {
            idx = "SSID" + (String)n;
            if(!prefs.remove(idx.c_str())) break;
            idx = "key" + (String)n;
            prefs.remove(idx.c_str());
            n++;
        }
        ....
```

**Comment by vkichline** [comment]
Thanks, Paul. I’d like to address this one first, because it affects most people and is simple.
I have a noob procedural question:
I forked M5ez a while back, made some changes, and merged them to master. I wish I had kept them in my own fork, but that boat has flown.
It don’t see any way to create another clean fork.
What’s the best way to submit just this delta, leaving the doc updates for second and sorting until later?
